reconcile/utils/ocm/ocm.py

```diff
@@ -502,20 +511,20 @@ def _get_create_cluster_spec(cluster_name: str, cluster: OCMSpec) -> dict[str, A
                     "sts": {
                         "enabled": True,
                         "auto_mode": True,
-                        "role_arn": cluster.spec.account.rosa.installer_role_arn,
-                        "support_role_arn": cluster.spec.account.rosa.support_role_arn,
+                        "role_arn": cluster.spec.account.rosa.sts.installer_role_arn,
+                        "support_role_arn": cluster.spec.account.rosa.sts.support_role_arn,
                         "instance_iam_roles": {
-                            "worker_role_arn": cluster.spec.account.rosa.worker_role_arn,
+                            "worker_role_arn": cluster.spec.account.rosa.sts.worker_role_arn,
                         },
                         "operator_role_prefix": f"{cluster_name}-{operator_roles_prefix}",
                     },
                 },
             }
 
-            if cluster.spec.account.rosa.controlplane_role_arn:
+            if cluster.spec.account.rosa.sts.controlplane_role_arn:
```

Comment:
will that fail is there is no `sts`?

Reply:
ah just saw `if not cluster.spec.account.rosa.sts:` above

Reply:
See line 503, an error will be raised before.